### DISCUSSION TITLE
Power Fx via Custom API

### DISCUSSION THREAD

**Discussion by filcole** [microsoft/Power-Fx#694] [Show and tell]
I've posted a blog explaining how one might use Power Fx in Dataverse using the dependent assemblies functionality. 
 It demonstrates creating a Custom API and using this to expose Power Fx to a cloud flow, but it could be anything that can talk to the dataverse API.

https://philcole.org/post/powerfx-customapi/

Thanks to @mikestall for helping solve the issue with using Power Fx as a dependent assembly; see the blog or https://github.com/microsoft/Power-Fx/issues/691

**Comment by MikeStall** [comment]
This is great!
